mcstatus/server.py

```diff
@@ -53,6 +53,7 @@ def status(self, retries=3, **kwargs):
                 pinger = ServerPinger(connection, host=self.host, port=self.port, **kwargs)
                 pinger.handshake()
                 result = pinger.read_status()
+                result.stripped_description = re.sub(r'\u00A7.', '', result.description)
```

Comment:
Currently not compatible with python 2 because of unicode vs. string

Reply:
Python 2 is officially depriciated now, so that's not a problem anymore